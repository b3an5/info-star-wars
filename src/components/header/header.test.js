import React from 'react'
import Header from './header.js'
import { shallow } from 'enzyme'

describe('header', () => {
  let wrapper 
  let mockSwitchCardCat = jest.fn()
  let mockSplash = false;

  it('should match snapshot', () => {
    wrapper = shallow(
      <Header 
        splash = {mockSplash}
        mockSwitchCardCat = {mockSwitchCardCat}
      />
    )
    expect(wrapper).toMatchSnapshot()
  })


})