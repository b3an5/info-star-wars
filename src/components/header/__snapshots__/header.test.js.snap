// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header should match snapshot 1`] = `ShallowWrapper {}`;
